_data/projects/OpinionatedGuides.yml

```
@@ -0,0 +1,11 @@
+name: Opinionated Guides
+desc: a one-stop guide to computers, linux, and electronics.
```

Minor change as this description will be listed on the site:

```suggestion
desc: A one-stop guide to computers, linux, and electronics.
```